spring-boot-admin-server/src/main/java/de/codecentric/boot/admin/web/servlet/resource/ConcatenatingResourceResolver.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2014 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package de.codecentric.boot.admin.web.servlet.resource;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.util.Collection;
+import java.util.Iterator;
+import java.util.List;
+
+import javax.servlet.http.HttpServletRequest;
+
+import org.springframework.core.io.Resource;
+import org.springframework.util.StreamUtils;
+import org.springframework.util.StringUtils;
+import org.springframework.web.client.ResourceAccessException;
+import org.springframework.web.servlet.resource.AbstractResourceResolver;
+import org.springframework.web.servlet.resource.ResourceResolver;
+import org.springframework.web.servlet.resource.ResourceResolverChain;
+
+import de.codecentric.boot.admin.io.resource.InMemoryFileResource;
+
+/**
+ * {@link ResourceResolver} that returns a resource with the concatenated resources given.
+ *
+ * @author Johannes Edmeier
+ */
+public class ConcatenatingResourceResolver extends AbstractResourceResolver {
+	private final byte[] delimiter;
+
+	public ConcatenatingResourceResolver(byte[] delimiter) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The user-supplied array 'delimiter' is stored directly.](https://www.codacy.com/app/joshiste/spring-boot-admin/file/2593395745/issues/source?bid=3175464&fileBranchId=3306528#l45)